src/helix-ui/styles/base/lists.less

```diff
@@ -1,5 +1,34 @@
+ul, ol {
+  list-style-type: none;
```

Comment:
The only downside to doing this is that you lose default browser support for other languages and `list-style-type` options. Not sure if thats out of scope.